pom.xml

```diff
@@ -177,6 +178,16 @@ limitations under the License.
                 <version>${org.slf4j.version}</version>
                 <scope>provided</scope>
             </dependency>
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-server</artifactId>
+                <version>${jetty.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-servlet</artifactId>
```

Comment:
Licensing looks okay for Jetty as it is dual licensed EPL & ASLv2, however there is a NOTICE at https://github.com/eclipse/jetty.project/blob/master/NOTICE.txt where I think we need to carry over the copyright in our NOTICE.